### DISCUSSION TITLE
域名搁着别浪费，Cloudflare + Gmail + Resend 十分钟轻松拥有免费的企业邮箱

### DISCUSSION THREAD

**Discussion by giscus** [CleanClip/CleanClipApp#18] [ArticleDiscussion]
# 域名搁着别浪费，Cloudflare + Gmail + Resend 十分钟轻松拥有免费的企业邮箱

现在的独立开发者们基本上是人手 N 个域名了，不过企业邮箱可能不是人人都有。

https://cleanclip.cc/zh/developer/cloudflare-worker-gmail-resend-enterprise-email/?utm_source=gapis.money

**Comment by yancheng2** [comment]
非常有用，感谢分享
